sites/skeleton.dev/src/components/examples/framework-components/steps/react/vertical.tsx

```diff
@@ -0,0 +1,38 @@
+import { Steps } from '@skeletonlabs/skeleton-react';
+
+const data = [
```

Comment:
nitpick:
```suggestion
const steps = [
```

Apply this everywhere if you're going to do it here.

Reply:
Renamed for all examples